flang/lib/Optimizer/Builder/Runtime/Command.cpp

```diff
@@ -0,0 +1,20 @@
+//===-- Command.cpp -- generate assignment runtime API calls ---------------===//
```

Comment:
Same nit as in command.h, too many --- and need another description.

Reply:
Done.